modules/local/seqera_runs_dump/functions.nf

```diff
@@ -49,8 +50,10 @@ Map getRunMetadata(meta, log, api_endpoint, trustStorePath, trustStorePassword)
             def workflowResponse = client.get(path: "/workflow/${runId}", query: ["workspaceId":workspaceId], headers: authHeader)
             if (workflowResponse.statusCode == 200) {
                 def metaMap = workflowResponse?.json?.workflow?.subMap("runName", "workDir", "projectName")
-                def config = new ConfigSlurper().parse( workflowResponse?.json?.workflow?.configText )
```

Comment:
This line fails to load in the config just for viralrecon but works for other pipelines.

A quick way to reproduce would be to download the raw `functions.nf` and append these at the end of the file. This would assume you have access to the workspaces below via exporting `TOWER_ACCESS_TOKEN` for Seqera Cloud.

```
// def meta = [id:'LanYfJk1vaHuD', workspace:'scidev/azure']         // WORKS: no fusion   / nf-core/rnaseq
// def meta = [id:'3Z627eCb3HqUy3', workspace:'seqeralabs/showcase'] // WORKS: with fusion / nf-core/rnaseq
// def meta = [id:'13gHs3OF4AYPI5', workspace:'scidev/azure']        // FAILS: no fusion   / nf-core/viralrecon
def api_endpoint = 'https://api.cloud.seqera.io/'
def java_truststore_path = ''
def java_truststore_password = ''
metaOut = getRunMetadata(meta, log, api_endpoint, java_truststore_path, java_truststore_password)
println(metaOut)
```

The fix in this PR is much more fragile because we are unable to load the config into a map and then checking whether Fusion is enabled. Instead, we are using a regex to check whether fusion is enabled via a pattern match in the  config string.

This isn't a great solution but we can't think of anything more robust right now and can make improvements in the future.